modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMFormFieldUtil.java

```diff
@@ -35,6 +36,16 @@ public static String getDDMFormFieldName(String ddmFormFieldName) {
 		return StringUtil.removeChar(ddmFormFieldName, CharPool.SPACE);
 	}
 
+	public static String getLegacyFieldName(String fieldName) {
```

Comment:
Please rename:
- `getLegacyFieldName` to `getLegacyDDMFormFieldName`
- `fieldName` to `ddmFormFieldName`